mautic/mautic
# [UX] ProTip to help marketers become power users

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds a template for ProTips that aim to provide extra information for users to get the most out of Mautic.

![image](https://github.com/user-attachments/assets/0f62c124-e9d9-4bee-9c28-77388930835f)

It's reusable and by default it accepts only a translation string for the tip, while icon and styles keep default for all.

It includes a total of 35 tips for listing pages.

---

### Tips

**Contacts**

- Use the <kbd>V</kbd> key to switch between card and table views 
- Add tags during import to help organize and segment your contacts. Example: (import_facebook_may2024) 
- Build your email list organically through opt-ins rather than purchasing lists 

**Forms**

-  Redirect URLs for forms can contain variables by using tokens like /thank-you?email={formfield=email} 
-  Minimize the number of fields to improve form submission rates. 
-  Use clear, action-oriented language in form headlines and CTAs (e.g. "Get your free guide" vs. "Submit"). 
- For longer forms, use a multi-step process to make them feel less overwhelming. 

**Tags**

- Tags can be used for quick segmentation, while custom fields are more suitable for long-term data storage. 
- Use tags instead of custom fields for multi-value contact attributes 
- You can create new tags while editing a contact. 

**Segments**

- Use behavioral data like website activity and email engagement to create dynamic segments that update automatically 
- Test different segment combinations to find the most effective groupings for your campaigns

**Assets**

- Gate valuable assets behind a form to capture lead information before allowing the download. 
- Use consistent naming conventions to keep assets organized
- Repurpose high-performing assets across different channels and campaigns
- Track asset performance metrics in the dashboard widgets to identify top-performing content

**Points**

- Implement a lead scoring decay model to accurately capture intent. 
- Reduce scores systematically over time if contacts become inactive. 
- Use a limit to prevent scores from getting too high or low 

**Triggers**

- Use behavioral triggers to send timely, relevant emails based on user actions. 

**Emails**

- Always include an unsubscribe link and never use a no-reply email address. 
- Preview and test your emails across devices and clients to ensure they look great everywhere. 
- Personalize email content as much as possible based on user behaviors and preferences 
- Avoid spam trigger words in subject lines and email content 
- Limit emails to one primary CTA to avoid decision paralysis 

**Pages**

- Optimize your website for mobile-first since most web traffic now comes from mobile devices 
- Place forms prominently "above the fold" so users don't have to scroll to find them. 

**Text messages**

- Keep messages concise and action-oriented to respect the personal nature of SMS 
- Use SMS for time-sensitive information like appointment reminders or limited-time offers 
- Comply with regulations by including opt-out instructions 

**Campaigns**

- Implement re-engagement campaigns for leads that become inactive 
- Deploy targeted survey campaigns to gather actionable insights and improve customer satisfaction 
- Use personalized cross-sell campaigns based on purchase history to increase customer lifetime value 
- Create automated onboarding sequences to guide new users through key product features and increase activation rates 

**Reports**

- Track DNC comments to reinsert contacts bounced by temporary issues. 







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the mentioned listing pages 
3. Scroll down to the bottom
4. See the tip
5. Reload the page to see another tip
